examples/water_boiler/.gitignore

```
@@ -1,2 +1,2 @@
-water_boiler_venv/ 
-
+venv/ 
+.vscode/
```

These changes seem unnecessary, especially since the README for the example specifically refers to `water_boiler_venv` as the name for the venv.

```suggestion
water_boiler_venv/
```